go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/dell/csi-powerstore
 
 go 1.17
 
+replace github.com/dell/gobrick => ../gobrick
```

Comment:
please remove reference to local directory

Reply:
@harshitap26 Please use the gonvme `main` branch commit id with `go get@<commit>`